code/game/machinery/computer/security.dm

```diff
@@ -570,6 +469,18 @@
 
 	..(severity)
 
+/**
+  * Sets a temporary message to display to the user
+  *
+  * Arguments:
+  * * text - Text to display, null/empty to clear the message from the UI
+  * * style - The style of the message: (color name), info, success, warning, danger
+  */
+/obj/machinery/computer/secure_data/proc/set_temp(text = "", style = "info", update_now = FALSE)
+	temp_notice = list(text = text, style = style)
+	if(update_now)
+		SStgui.update_uis(src)
+
 /obj/machinery/computer/secure_data/detective_computer
 	icon = 'icons/obj/computer.dmi'
 	icon_state = "messyfiles"
```

Comment:
While youre working in this file, can you remove this? It has an awful icon override, and that state doesnt even exist. Its also not pathed into any maps